executor/executor_test.go

```diff
@@ -91,17 +104,32 @@ func (d *dummySDAPIBroken) PostCommand(specPath string, smallSpec *util.CommandS
 	return nil, fmt.Errorf("Something error happen")
 }
 
-func dummyAPICommand(format string) (cmd *util.CommandSpec) {
-	cmd = &util.CommandSpec{
+func dummyAPICommandBinary(format string) (cmd *util.CommandSpec) {
```

Comment:
These `CommandSpec` factory functions are redundant. We should return an appropriate `CommandSpec` object by the `format` parameter.

Reply:
I misunderstood about this function. I fixed it.